common-util/src/main/java/org/sunbird/common/responsecode/ResponseMessage.java

```diff
@@ -317,6 +318,8 @@
     String CASSANDRA_CONNECTION_ESTABLISHMENT_FAILED =
         "Cassandra connection establishment failed in {0} mode.";
     String COMMON_ATTRIBUTE_MISMATCH = "{0} mismatch of {1} and {2}";
+    String DUPLICATE_EXTERNAL_ID =
```

Comment:
Remove this error message. Consider ADD as UPSERT if external ID exists for same user for given id type and provider. This is to simply usage of update user API.